docs-src/content/getting-started/cubic_docker.md

```diff
@@ -17,42 +17,44 @@ It is not necessary to go through through these steps to call the demo server fo
     IMAGE_NAME="name you want to give the file (should end with the same extension as the url, usually bz2)"
     curl $URL -L -o $IMAGE_NAME
     ```
-   
-2. Load the docker image: 
+
+3. Load the docker image
 
     ```bash
     docker load < $IMAGE_NAME
     ```
 
     This will output the name of the image (e.g. cubicsvr-demo-en_us-16).
 
-3. Start the cubic service listening:
+4. Start the cubic service
 
     ```bash
     docker run -p 2727:2727 -p 8080:8080 --name cobalt cubicsvr-demo-en_us-16
     ```
 
     That will start listening for grpc commands on port 2727 and http requests on 8080, and will stream the debug log to stdout.  (You can replace `--name cobalt` with whatever name you want.  That just provides a way to refer back to the currently running container.)
 
-4. Verify the service is running by calling 
+5. Verify the service is running by calling
 
     ```bash
     curl http://localhost:8080/api/version
     ```
 
-5. If you want to explore the package to see the model files etc, call the following to open the bash terminal on the previously run image.  Model files are located in the /model directory.
-   
+6. If you want to explore the package to see the model files etc, call the following to open the bash terminal on the previously run image.  Model files are located in the /model directory.
+
     ```bash
     docker exec -it cobalt bash
     ```
-   
+
 ## Contents of the docker image
+
 - **Base docker image** : debian-stretch-slim
 - **Additional dependencies** (installed with yum install on centos or apt-get on ubuntu)
```

Comment:
We should nemove the paranthesis. It's already installed in the image, and might confuse readers here wrt whether they need to do something.